tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "@tsconfig/svelte/tsconfig.json",
+  "compilerOptions": {
+    "target": "ESNext",
+    "useDefineForClassFields": true,
+    "module": "ESNext",
+    "resolveJsonModule": true,
+    /**
+     * Typecheck JS in `.svelte` and `.js` files by default.
+     * Disable checkJs if you'd like to use dynamic types in JS.
+     * Note that setting allowJs false does not prevent the use
+     * of JS in `.svelte` files.
+     */
+    "allowJs": true,
+    "checkJs": true,
+    "isolatedModules": true,
+    "types": ["vite/client"]
+  },
+  "include": ["*.js", "*.ts", "*.svelte", "components/**/*.js", "components/**/*.ts", "components/**/*.svelte", "tests/*"]
```

Comment:
This is really important to get right; I got it slightly wrong at first and couldn't figure out why there weren't errors